exercises/4-an-enslaved-hope/README.md

```diff
@@ -45,171 +45,113 @@ _____
-```groovy
-    environment {
-        // GLobal Vars
-        PIPELINES_NAMESPACE = "<YOUR_NAME>-ci-cd"
-        APP_NAME = "todolist-api"
-
-        JENKINS_TAG = "${JOB_NAME}.${BUILD_NUMBER}".replace("/", "-")
-        JOB_NAME = "${JOB_NAME}".replace("/", "-")
-
-        GIT_SSL_NO_VERIFY = true
-        GIT_CREDENTIALS = credentials('jenkins-git-creds')
-        GITLAB_DOMAIN = "gitlab.<APPS_URL>"
-        GITLAB_PROJECT = "<GIT_USERNAME>"
-    }
-```
-
-4. With these changes in place, push your changes to the `feature/jenkinsfile` branch.
-```bash
-git add Jenkinsfile
-```
-```bash
-git commit -m "ADD - namespace and git repo to pipeline"
-```
-```bash
-git push
-```
-
-5. When the changes have been successfully pushed; Open Jenkins.
-
-6. Create a `New Item` on Jenkins. Give it the name `todolist-api` and select `Multibranch Pipeline` from the bottom of the list as the job type.
-![multibranch-select](../images/exercise4/multibranch-select.png)
-
-7. On the job's configure page; set the Branch Sources to `git`
-![multibranch-select-git](../images/exercise4/multibranch-select-git.png)
-
-8. Fill in the Git settings with your `todolist-api` GitLab url and set the credentials as you've done before. `https://gitlab.<APPS_URL>/<YOUR_NAME>/todolist-api.git`
-![multibranch-git](../images/exercise4/multibranch-git.png)
-
-9. Set the `Scan Multibranch Pipeline Triggers` to be periodic and the interval to 1 minute. This will poll the GitLab instance for new branches or change sets to build.
-![multibranch-scan-time](../images/exercise4/multibranch-scan-time.png)
-
-10. Save the Job configuration to run the intial scan. The log will show scans for `master` and `develop` branches, which have no `Jenkinsfile` so are skipped. The resulting view will show the `feature/jenkinsfile` job corresponding the only branch that currently has one. The build should run automatically.
-![todolist-api-multi](../images/exercise4/todolist-api-multi.png)
+### Part 1 - Security Scanning Slaves
+> _This part of the exercise focuses on updating the `enablement-ci-cd` repo with some new jenkins-slave pods for use in future exercise_
 
-11. The pipeline file is setup to only run `bake` & `deploy` stages when on `master` or `develop` branch. This is to provide us with very fast feedback for team members working on feature or bug fix branches. Each time someone commits or creates a new branch a basic build with testing occurs to give very rapid feedback to the team. Let's now update our  `master` and `develop` branches to include the Jenkinsfile and delete the feature branch.
-```bash
-git checkout develop
-```
-```bash
-git merge feature/jenkinsfile
-# you may get merge conflicts at this point
-```
-> NOTE: You may see a number of merge conflict warnings at this point. You can safely ignore them and proceed to commit the changes.
+<!-- #### 3a - OWASP ZAP
```

Comment:
3a -> 1a?

Reply:
That's in a comment, as I think GLS removed it. Probs worth plugging it back in but verifying it's working as expected first